modular_skyrat/modules/reagent_forging/code/anvil.dm

```diff
@@ -15,50 +16,46 @@
 		icon_state = "primitive_anvil_empty"
 
 /obj/structure/reagent_anvil/attackby(obj/item/I, mob/living/user, params)
-	var/obj/item/forging/incomplete/searchIncompleteSrc = locate(/obj/item/forging/incomplete) in contents
-	if(istype(I, /obj/item/forging/hammer) && searchIncompleteSrc)
+	var/obj/item/forging/incomplete/search_incomplete_src = locate(/obj/item/forging/incomplete) in contents
```

Comment:
This variable makes it feel like all the times its used is in searching, not using. Try something like `./forging/incomplete/found_incomplete` for stuff like this so we know we're using the found one from the variable's definition